SUMMARY.md

```diff
@@ -35,6 +35,8 @@
   * [Lean frontend assets: Apostrophe without jQuery](tutorials/core-concepts/front-end-assets/lean-frontend-assets.md)
 * [Users and Permissions](tutorials/core-concepts/users-and-permissions/README.md)
   * [Users and Groups](tutorials/core-concepts/users-and-permissions/users-and-groups.md)
+  * [Managing Access Control](tutorials/core-concepts/users-and-permissions/managing-accesss-control.md)
+  * [OpenAuth with Apostrophe Passport](tutorials/core-concepts/users-and-permissions/oauth-passport.md)
```

Comment:
OpenAuth is a specific product we're not using here. Should read "Single Sign-On with Apostrophe Passport."

Reply:
(I assume. Unless you actually used OpenAuth? But it is still not a title that seems immediately relevant if you're not using that particular thing.)

Reply:
Removed all incorrect references to oauth/OpenAuth.